com.surfshark.Surfshark.yaml

```diff
@@ -0,0 +1,74 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+app-id: com.surfshark.Surfshark
+runtime: org.gnome.Platform
+runtime-version: '45'
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk: org.gnome.Sdk
+command: surfshark
+separate-locales: false
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+modules:
+  - 'libndp.yaml'
+  - 'network-manager.yaml'
+  - 'libnma.yaml'
+  - 'network-manager-openvpn.yaml'
+  - name: surfshark
+    buildsystem: simple
+    build-commands:
+      - ar -x surfshark_*.deb
+      - rm -f surfshark_*.deb
+      - tar -xf data.tar.xz
+
+      - install surfshark $FLATPAK_DEST/bin/
+      - install -D "usr/share/applications/surfshark.desktop" "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - install -D "usr/share/icons/hicolor/128x128/apps/surfshark.png" "${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png"
+      - install -D "usr/share/icons/hicolor/256x256/apps/surfshark.png" "${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png"
```

Comment:
```suggestion
```

Only installing the largest one is fine.

Reply:
updated, 512x512 is the only exposed icon